### PR TITLE
Some more renaming from "race" to "faction"

### PR DESCRIPTION
**Pull request by penev92**
Yet another piece of #7873.
Split into nice small commits for easy reviewing.

### CODE REVIEW

File: OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs [hunk 1]
```diff
@@ -12,6 +12,7 @@
 using System.Collections.Generic;
 using System.Globalization;
 using System.Linq;
+using OpenRA.Scripting;
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs]: This looks like an accidental change.

**Comment by abcdefg30** [comment]
Looks good to me otherwise. :+1: / :white_check_mark:

**Comment by penev92** [comment]
Fixed.

**Comment by pchote** [comment]
:+1:

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/3e830b3c47931713ff155eaf35b70e6bb381da9b%5E...3e830b3c47931713ff155eaf35b70e6bb381da9b

**Comment by penev92** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/2870d06a00486575aab336a89980fe25445a3306...5eb464bce4f4040ac6382d066df97d2f649eb97b

**Pull request merged**